### ISSUE TITLE
Update all references to old repo location to the new one 

### ISSUE THREAD

**Issue by deivid-rodriguez** [rubygems/rubygems#3264]
We moved the bundler repo from it's old location: https://github.com/bundler/bundler to the rubygems organization: https://github.com/rubygems/bundler.

We should update all references inside the repository to point to the new location.

**Comment by deivid-rodriguez** [comment]
@duckinator Txt man pages are going away in https://github.com/rubygems/rubygems/pull/3921, so you should be unblocked now.

**Comment by duckinator** [comment]
@deivid-rodriguez good to know! I also just realized that it got merged into the rubygems/rubygems repo since this issue was opened, so URLs may need to be updated again. :sweat_smile:

**Comment by deivid-rodriguez** [comment]
Indeed, I think most stuff is done by now though. From your table in https://github.com/rubygems/rubygems/issues/3264#issuecomment-576051611, I think the changelog can be skipped, since references to PRs don't really translate accross repos.
